rules/consistent-phone-message/rule.md

```diff
@@ -1,25 +1,29 @@
 ---
 type: rule
-archivedreason: 
 title: Do you use a consistent phone message?
-guid: fef66d03-eb72-4e71-a0af-d239ddc15d94
 uri: consistent-phone-message
-created: 2015-11-24T18:42:48.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
 related: []
 redirects:
-- do-you-use-a-consistent-phone-message
-
+  - do-you-use-a-consistent-phone-message
+created: 2015-11-24T18:42:48.000Z
+archivedreason: null
+guid: fef66d03-eb72-4e71-a0af-d239ddc15d94
 ---
+You should remember that any point of contact leaves an impression on clients or customers so even the answering service on your phones should be taken into consideration. 
 
-You should remember that any point of contact leaves an impression on clients or customers so even the answering service on your phones should be taken into consideration. If you have a mobile phone which is used for work purposes.
+Your personal mobile phone should have a standard answering message for any missed calls.  Many people don't use voice mail these days, so you may as well tell people about it. Also, if you don't mind it is nice to include subtle company branding.
 
 <!--endintro-->
 
-It's recommended that you use a standard message like this:
+> "Hi, you've reached Adam Cogan. If you can leave your name, number and a short message I will call you back as soon as I can. Thanks!"
+
+**Figure: Bad example**
```

Comment:
cc @pierssinclairssw 

Hey @daviddangssw,

Would it be better to use the good and bad figures here or is it not needed? See Below for examples
Check out this [rule](https://www.ssw.com.au/rules/figures-do-you-use-bad-example-and-good-example-with-ticks-and-crosses-in-captions)
```
::: good
Figure: Good example 
:::
```
 
 ```
::: bad
Figure: Bad example 
:::
```